### PR TITLE
Spelling

### PR DESCRIPTION
**Pull request by jsoref**
Fixes misspellings identified by the [check-spelling action](https://github.com/marketplace/actions/check-spelling).

The misspellings have been reported at https://github.com/jsoref/qdrant/actions/runs/6175154760#summary-16761388526

The action reports that the changes in this PR would make it happy: https://github.com/jsoref/qdrant/actions/runs/6175155473#summary-16761390902

### All Submissions:

* [x] Contributions should target the `dev` branch. Did you create your branch from `dev`?
* [x] Have you followed the guidelines in our Contributing document?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../../../pulls) for the same update/change?

### Changes to Core Features:

* [ ] Have you added an explanation of what your changes do and why you'd like us to include them?
* [ ] Have you written new tests for your core changes, as applicable?
* [ ] Have you successfully ran tests with your changes locally?

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -6,11 +6,11 @@ We love your input! We want to make contributing to this project as easy and tra
 - Submitting a fix
 - Proposing new features
 
-## We Develop with Github
+## We Develop with GitHub
```
Comment (jsoref) [inline CONTRIBUTING.md:9]: brand

File: docs/grpc/docs.md [hunk 2]
```diff
@@ -2935,7 +2935,7 @@ For example, if `oversampling` is 2.4 and `limit` is 100, then 240 vectors will
 | Search | [SearchPoints](#qdrant-SearchPoints) | [SearchResponse](#qdrant-SearchResponse) | Retrieve closest points based on vector similarity and given filtering conditions |
 | SearchBatch | [SearchBatchPoints](#qdrant-SearchBatchPoints) | [SearchBatchResponse](#qdrant-SearchBatchResponse) | Retrieve closest points based on vector similarity and given filtering conditions |
 | SearchGroups | [SearchPointGroups](#qdrant-SearchPointGroups) | [SearchGroupsResponse](#qdrant-SearchGroupsResponse) | Retrieve closest points based on vector similarity and given filtering conditions, grouped by a given field |
-| Scroll | [ScrollPoints](#qdrant-ScrollPoints) | [ScrollResponse](#qdrant-ScrollResponse) | Iterate over all or filtered points points |
+| Scroll | [ScrollPoints](#qdrant-ScrollPoints) | [ScrollResponse](#qdrant-ScrollResponse) | Iterate over all or filtered points |
```
Comment (jsoref) [inline docs/grpc/docs.md:2938]: This is a duplicated word as opposed to a true misspelling, but spell checkers have flagged such things for decades.

File: docs/redoc/v0.10.0/openapi.json [hunk 3]
```diff
@@ -3229,7 +3229,7 @@
         }
       },
       "NamedVectorStruct": {
-        "description": "Vector data separator for named and unnamed modes Unanmed mode:\n\n{ \"vector\": [1.0, 2.0, 3.0] }\n\nor named mode:\n\n{ \"vector\": { \"vector\": [1.0, 2.0, 3.0], \"name\": \"image-embeddings\" } }",
+        "description": "Vector data separator for named and unnamed modes Unnamed mode:\n\n{ \"vector\": [1.0, 2.0, 3.0] }\n\nor named mode:\n\n{ \"vector\": { \"vector\": [1.0, 2.0, 3.0], \"name\": \"image-embeddings\" } }",
```
Comment (jsoref) [inline docs/redoc/v0.10.0/openapi.json:3232]: This probably comes from here: https://github.com/qdrant/qdrant-web-ui/blob/5bf5f68291fa5a69b1021e7d57cd41f792662b47/public/openapi.json#L4171
And thus should probably be dropped and fixed there...

File: docs/redoc/v0.4.2/openapi.json [hunk 4]
```diff
@@ -1737,7 +1737,7 @@
     }
   },
   "externalDocs": {
-    "description": "Find out more about Qdrant aplications and demo",
+    "description": "Find out more about Qdrant applications and demo",
```
Comment (jsoref) [inline docs/redoc/v0.4.2/openapi.json:1740]: I can't find any other source for this text... https://grep.app/search?q=Find%20out%20more%20about%20Qdrant%20aplications%20and%20demo
https://github.com/search?q=Find+out+more+about+Qdrant+aplications+and+demo+NOT+repo%3Aqdrant%2Fqdrant+NOT+is%3Afork&type=code

File: docs/redoc/v0.10.0/openapi.json
Comment (timvisee) [inline docs/redoc/v0.10.0/openapi.json:1]: Could you drop all changes for these files: `docs/redoc/*`?

I don't think we can just change these old definitions.

The core problem is already fixed. Newly generated definitions don't have this spelling issue anymore:

https://github.com/qdrant/qdrant/blob/a6164ccda89b00eb361686ed40b90da9fbb1b2c3/docs/redoc/master/openapi.json#L5700-L5702

**Review by timvisee — CHANGES_REQUESTED** [review]

File: docs/redoc/v0.10.0/openapi.json (hunk 3 above)
Comment (timvisee) [reply docs/redoc/v0.10.0/openapi.json:3232]: It is sourced from here, and already fixed:
https://github.com/qdrant/qdrant/blob/a6164ccda89b00eb361686ed40b90da9fbb1b2c3/docs/redoc/master/openapi.json#L5700-L5702

File: docs/redoc/v0.10.0/openapi.json
Comment (timvisee) [reply docs/redoc/v0.10.0/openapi.json:1]: Thanks! :raised_hands:

**Review by timvisee — APPROVED** [review]
Thank you for your efforts on this.

**Pull request merged**